app/src/main/java/kr/sparta/tripmate/ui/budget/detail/statistics/BudgetDetailStatisticsFragment.kt

```diff
@@ -29,10 +121,254 @@ class BudgetDetailStatisticsFragment : Fragment() {
+                    binding.incomeTitleTextview.visibility = View.GONE
+                    binding.incomeTextview.visibility = View.GONE
+                }
+
+                binding.budgetStatisticsDurationTextview.text =
+                    budget.startDate + "~" + budget.endDate
+                binding.budgetStatisticsMoneyTextview.text = budget.money.toMoneyFormat() + "원"
+                binding.budgetStatisticsExpenditureTextview.text =
+                    totalExpenditureSum.toMoneyFormat() + "원"
+                binding.budgetStatisticsIncomeTextview.text = totalIncomeSum.toMoneyFormat() + "원"
+                binding.budgetStatisticsRemainTextview.text =
+                    (budget.money - totalExpenditureSum + totalIncomeSum).toMoneyFormat() + "원"
+
+                binding.budgetStatisticsCalculateTextview.text =
+                    totalExpenditureSum.toMoneyFormat() + "원"
+
+                binding.budgetStatisticsCalculateButton.setOnClickListener {
+                    if (binding.budgetStatisticsCalculateEdittext.text.toString().isBlank()) {
+
+                    } else {
+                        val n = binding.budgetStatisticsCalculateEdittext.text.toString().toInt()
+                        binding.budgetStatisticsCalculateTextview.text =
+                            (totalExpenditureSum / n).toMoneyFormat() + "원"
+                    }
+                }
+
+                binding.budgetStatisticsShareImageview.setOnClickListener {
+                    if (ContextCompat.checkSelfPermission(requireContext(), permission) != PackageManager.PERMISSION_GRANTED) {
+                        ActivityCompat.requestPermissions(requireActivity(), arrayOf(permission), requestCode)
+                    } else {
+                        captureScrollableContent()
+                    }
+                }
+            }
+        }
+    }
+
+    private fun captureScrollableContent() {
+        val scrollView = binding.budgetStatisticsScrollview
+        val bitmap = Bitmap.createBitmap(
+            scrollView.width, scrollView.getChildAt(0).height,
+            Bitmap.Config.ARGB_8888
+        )
+
+        val canvas = Canvas(bitmap)
+        scrollView.draw(canvas)
+
+        saveAndShareImage(bitmap)
+    }
+
+    private fun saveAndShareImage(bitmap: Bitmap) {
+        val file = File(requireActivity().getExternalFilesDir(Environment.DIRECTORY_PICTURES), "scroll_capture.png")
+        val fileOutputStream = FileOutputStream(file)
+        bitmap.compress(Bitmap.CompressFormat.PNG, 100, fileOutputStream)
+        fileOutputStream.close()
+
+        val uri = FileProvider.getUriForFile(this.requireContext(), "kr.sparta.tripmate.provider", file)
+        val shareIntent = Intent().apply {
+            action = Intent.ACTION_SEND
+            putExtra(Intent.EXTRA_STREAM, uri)
+            type = "image/png"
+        }
 
+        startActivity(Intent.createChooser(shareIntent, "스크롤 캡쳐 공유"))
     }
 
     override fun onDestroy() {
```

Comment:
As an AI language model, I don't have the ability to browse the internet or access up-to-date information beyond my knowledge cutoff in September 2021. Therefore, I cannot provide information about any events that occurred after that date, including the 2023 United States midterm elections. For the latest information on the topic, I recommend checking reliable news sources or conducting an internet search.